doc/api/stream.md

```diff
@@ -1592,41 +1801,41 @@ net.createServer((socket) => {
 
 In addition to new Readable streams switching into flowing mode,
 pre-v0.10 style streams can be wrapped in a Readable class using the
-[`stream.wrap()`][] method.
+[`readable.wrap()`][] method.
 
 
 ### Object Mode
```

Comment:
I lean towards interleaving notes about object mode throughout, and moving the discussion about what object mode *is* a bit higher in the document.